You_or_ME

manognyaa opened the pull request:
## Related Issue 
Since Lockdown started, Everyone is feeling bored. So, I thought why not create a 2 player game ?
so, I have created this game, where two persons can play.
- Info about the related issue 

- [x] LGM-SOC'21 Participant 
- [ ] Contributor

Closes: #226  

### Describe the changes you've made
I have created this game with the help of numpy, math, sys and pygame. 
## Type of change

What sort of change have you made:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?
Once you download this game in your local computer, it will run successfully.

## Checklist:
<!--
Example how to mark a checkbox:-
- [x] My code follows the code style of this project.
-->
- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly whereever it was hard to understand.
- [x] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots

 Original           | Updated
 :--------------------: |:--------------------:
![game 1](https://user-images.githubusercontent.com/77045147/121992806-fd79d600-cdbf-11eb-9fee-a53809efb9ec.png)
![game 2](https://user-images.githubusercontent.com/77045147/121992815-01a5f380-cdc0-11eb-9799-957f3ace4300.png)
![game _you or me](https://user-images.githubusercontent.com/77045147/121992832-08346b00-cdc0-11eb-888f-f91bf0fb2b68.gif)

 | Updated Screenshot |

@prathimacode-hub @Debashish-hub
If there any changes required, please guide me

Debashish-hub commented:
Do comment your code wherever possible
Rename your folder to You or Me
Add the gifs to the readme file 

@manognyaa

manognyaa commented:
> Do comment your code wherever possible
> Rename your folder to You or Me
> Add the gifs to the readme file
> 
> @manognyaa

yes sir, I have added the gif to readme file and added comments wherever required. also updated the folder name as You or Me.
@Debashish-hub @prathimacode-hub  pls guide me, if any changes required.

Debashish-hub commented:
Good job. Your PR is approved @manognyaa 
Keep it up
